test/util/cluster/cluster.go

```diff
@@ -19,6 +19,28 @@ import (
 	"github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/pod"
 )
 
+func init() {
```

Comment:
maybe we shall call the init from sriovv1 like this: https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/155/files#diff-fa9c480471555909f4af5c083bdc918e887e487b12ab7eb49cc27bf10757bb0cR53 ?

Reply:
Remove `init()`, as it has been solved by #155.